Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -19,6 +19,10 @@ function HandleCodeAnalysisReporting
         
         PostAndResolveComments $messages "SonarQube Code Analysis"
     }	
+    else
+    {
+        Write-Verbose "The build was not triggered by a Pull Request, not processing code analysis comments"   
```

Comment:
Unrelated change - there should be a log message in the build is not a PR build to diagnose potential issues.